server/appbackend/serializers.py

```diff
@@ -3,27 +3,35 @@
 from .models import *
 
 
+class PersonSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Person
+        fields = '__all__'
+
+
 class StudentSerializer(serializers.ModelSerializer):
-    role = serializers.CharField(source='role.name')
 
     class Meta:
         model = Person
-        fields = ('id', 'name', 'phone', 'role')
+        fields = ('id', 'first_name', 'last_name', 'phone')
```

Comment:
Why change from name to "first name and last name"?

Reply:
Default django User fields, already are implemented. If the front end prefers to use only name and split it before sending to back end, we can do that.